corehq/apps/cloudcare/static/cloudcare/js/backbone/cases.js

```diff
@@ -283,23 +293,22 @@ cloudCare.CaseListView = Backbone.View.extend({
                 "sSearch": "Filter cases:",
                 "sEmptyTable": "No cases available. You must register a case to access this form."
             },
-            sScrollX: $('#case-list').css('width'),
+            sScrollX: +$('#case-list').css('width').replace('px', ''),
```

Comment:
I think it's a better habit to use `.width()`, `.outerWidth()`, etc. rather than pulling width out of CSS, because the former is more generally applicable.